Source/Urho3D/AngelScript/ScriptInstance.cpp

```diff
@@ -643,7 +643,7 @@ void ScriptInstance::GetScriptAttributes()
     {
         const char* name;
         int typeId;
-        bool isPrivate, isProtected, isHandle;
+        bool isPrivate, isProtected, isHandle, isEnum;

```

Comment:
Uninitialized variables suck.
Please, initialize them _all_ in these three lines.